qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -554,13 +560,6 @@ def __eq__(self, other: "SymbolicPulse") -> bool:
 
         return True
 
-    def __hash__(self) -> int:
```

Comment:
Should we note this removal in the release notes?

Reply:
Good point. Should be noted as upgrade.

Reply:
Thank you.